cmd/brew-bundle.rb

```diff
@@ -61,8 +63,12 @@
   exit 0
 end
 
+# Pop the named command from ARGV, leaving everything else in place
+command = ARGV.named.first
+i = ARGV.find_index(command)
+ARGV.slice!(i)
```

Comment:
You probably want to make it `ARGV.delete_at(ARGV.index(command) unless command.nil?` to handle the case when command is nil.